source/collaborate/convert-group-messages.rst

```diff
@@ -0,0 +1,18 @@
+Convert group messages to private channels
+==========================================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+You can change the members of your group conversation by converting the group message to a private channel. When a group message is converted to private, its history and membership are preserved. Membership in a private channel remains as invitation only.
+
+.. note::
+
+    - Any member of an existing group message can convert that group message to a private channel, but all group message particpants must be members of the same team.
+    - You can convert a group message to a private channel using a web browser or the desktop app.
+
+To convert a group message to a private channel:
+
+1. Select the group message name at the top of the center pane to access the drop-down menu, then select **Convert to Private Channel**.
+2. Enter a channel name.
```

Comment:
A possible second step is to also select the team you want to the private channel to go into. When the GM members have multiple common teams between them, the user needs to select the team.